src/main/java/org/sagebionetworks/web/client/security/AuthenticationControllerImpl.java

```diff
@@ -50,6 +51,16 @@ public class AuthenticationControllerImpl implements AuthenticationController {
     "last_user_authentication_receipt";
   private static final String AUTHENTICATION_MESSAGE =
     "Invalid username or password.";
+  public static final String NIH_NOTIFICATION_DISMISSED =
+    "nih_notification_dismissed";
+  public static String COOKIES_ACCEPTED =
+    "org.sagebionetworks.security.cookies.notification.okclicked";
+
+  private String[] persistentLocalStorageKeys = new String[] {
+    USER_AUTHENTICATION_RECEIPT,
+    NIH_NOTIFICATION_DISMISSED,
+    COOKIES_ACCEPTED,
+  };
```

Comment:
There may be other keys referenced in SRC only that we want to persist

Reply:
That's true.  Today, SRC values in local storage are cleared on logout.

Reply:
This can be extended to add those SRC keys that we want to protect